### ISSUE TITLE
[Suggestion] cglow block using fill command syntax

### ISSUE THREAD

**Issue by AliahX** [Earthcomputer/clientcommands#291]
I would like to be able to specify two coordinates when using cglow and have the replace feature.

**Comment by barraIhsan** [comment]
You can specify 2 coordinates by using `cglow area <x> <y> <z> <x1> <y1> <z1>` iirc

**Comment by AliahX** [comment]
Cool! I didn't know about this, however i still feel that there should be replace syntax

**Comment by barraIhsan** [comment]
what replace syntax for? It's a client-side mod. It can't replace a block.

**Comment by OneYellowLemon** [comment]
I would assume that `replace` is referring to the `replace` syntax of the vanilla fill command to only affect certain blocks. `replace` is not a good name for this use case, perhaps `filter` would fit better. However, the only use I can really see is quickly highlighting certain blocks and I feel this will (almost) always be used for x-ray...

**Comment by barraIhsan** [comment]
> always be used for x-ray...

you also can do it with `cfindblock`. Also, `filter` will be great. A feature can be bad depends who use it.

**Comment by OneYellowLemon** [comment]
Yes, but then you at least have to run the command each time you want to find a new block and then click the thing in chat to highlight it. Not hard to automate it with the scripting API or anything, but certainly a deterrent. With a filter, you could easily do something like `/cglow area ~-100 0 ~-100 ~100 255 ~100 diamond_ore` and x-ray super easily.

**Comment by AliahX** [comment]
or you could literally just get an xray texturepack? Its not that hard.

**Comment by xpple** [comment]
This could be easily added to `/careastats`, which I would find more fitting.

**Comment by RealRTTV** [comment]
isn't the fill replace feature already implemented?

**Comment by Earthcomputer** [comment]
> isn't the fill replace feature already implemented?

For ghost blocks yes but that's not the same as the suggestion
